src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -391,18 +390,6 @@ public List<Assessment> getAssessments(@PathVariable("bookingId") @Parameter(des
         return inmateService.getAssessments(bookingId);
     }
 
-    @ApiResponses({
-        @ApiResponse(responseCode = "200", description = "OK"),
-        @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
-        @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
-        @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
-    @Operation(summary = "Image detail (without image data).", description = "Image detail (without image data).")
-    @GetMapping("/{bookingId}/image")
```

Comment:
unused

Reply:
I wonder how it ended up on `@SlowReportQuery` if it's not being used.

Reply:
hmmm interesting point - most seem to the image by offenderNo now so possibly before that happened.